src/CarbonAware.Aggregators/test/CarbonAwareAggregatorTests.cs

```diff
@@ -344,6 +343,7 @@ public async Task<double> CalculateAverageCarbonIntensityAsync_ValidTimeInterval
 
     [TestCase("2021-11-18T00:00:00Z", null, TestName = "AverageCI valid startTime, null endTime")]
     [TestCase(null, "2021-11-18T00:00:00Z", TestName = "AverageCI null startTime, valid endTime")]
+    [TestCase("2021-11-19T00:00:00Z", "2021-11-18T00:00:00Z", TestName = "AverageCI startTime greater than endTime")]
     public void TestCalculateAverageCarbonIntensityAsync_InvalidStartEndTimes_ThrowsException(string start, string end)
```

Comment:
```suggestion
    public void CalculateAverageCarbonIntensityAsync_InvalidStartEndTimes_ThrowsException(string start, string end)
```

Reply:
No need for the test moniker, I think.

Reply:
agree "in spirit" @KanishkT123 -- and since the "Test" prefix is used throughout this test file, we followed the pattern

Reply:
after a chat with bderusha, will open a follow-up clean-up story